### ISSUE TITLE
vaadin-context-menu-flow 1.2.1 using 1.2-SNAPSHOT for parent pom

### ISSUE THREAD

**Issue by maaxiim** [vaadin/platform#458]
Good morning, 

I noticed 12.0.1 was released yesterday and so I updated my vaadin BOM to 12.0.1 and I am getting the following build error 

```
[ERROR] Failed to execute goal on project NAME: 
Could not resolve dependencies for project NAME:jar:2.0.1-SNAPSHOT: 
Failed to collect dependencies at 
com.vaadin:vaadin:jar:12.0.1 -> 
com.vaadin:vaadin-core:jar:12.0.1 -> 
com.vaadin:vaadin-grid-flow:jar:2.1.3 -> 
com.vaadin:vaadin-context-menu-flow:jar:1.2.1: 
Failed to read artifact descriptor for com.vaadin:vaadin-context-menu-flow:jar:1.2.1: 
Could not find artifact com.vaadin:flow-component-base:pom:1. 2-SNAPSHOT in snapshot-repos
```

**Comment by maaxiim** [comment]
wrong owning project - please delete

**Comment by alvarezguille** [comment]
Fixed in [12.0.2](https://github.com/vaadin/platform/releases/tag/12.0.2)
